auth-web/src/util/constants.ts

```diff
@@ -247,6 +247,48 @@ export enum NrEntityType {
     INFO = 'INFO', // special value for sub-menu
 }
 
+export enum NrRequestType {
```

Comment:
I had to guess a bit at which of the [enum NR request types](https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/name-request-codes.ts) corresponded with the request types in the [ux pin](https://preview.uxpin.com/a86dccdd20ac62828f965c5eea10f81b828a9853#/pages/163299967/simulate/sitemap). There are also a couple types I can across when testing that I am not sure about, so the code will default to not displaying a type (for now). These codes I am unsure about are: DBA, GP, XCP.

Reply:
Nice digging to find the shared enum!

Does this help?
https://github.com/bcgov/namerequest/blob/feature-way-of-navigating/src/enums/entity-type.ts

Reply:
This app isn't consistent with enum naming, but it should be plural.

Reply:
Sev, that link helps me understand what each code means, but I'm still not sure how the entity codes map onto the NR types from the ux pin: Registration, Incorporation, Continuation In, Reactivation, Amalgamation, Name Change, and Alteration.

Also I'll make the enum plural.

Reply:
These look good to me but I'm not an expert. I recommend you confirm these with Mihai.

Also, these should be called out in your ticket under a "Test Notes" comment so that QA knows what to verify.

Reply:
OK, here's another link for ya:
https://github.com/bcgov/namex/blob/7e5c88a86a0585a404bafc549a8755b74264b06a/api/namex/services/lookup/name_request_filing_actions.py#L51

Reply:
@chenhongjing -^ samething I sent you lol

Reply:
I'm OK to resolve this conversation if you are.